x-pack/plugins/transform/public/app/common/transform.ts

```diff
@@ -78,3 +78,9 @@ export const useRefreshTransformList = (
     },
   };
 };
+
+export const overrideTransformForCloning = (originalConfig: TransformConfigUnion) => {
+  // 'Managed' means job is preconfigured and deployed by other solutions
+  // we should not clone this setting
+  return { ...originalConfig, _meta: { ...originalConfig._meta, managed: false } };
```

Comment:
Nit - should we be adding
```
  "_meta": {
    "managed": false
  }
```
when cloning any transform (including ones which aren't `managed`), or not adding the `managed` field at all?

Reply:
The problem with setting it to `false` is that we'll only get transforms with `"managed": false` in them if they are cloned. 
The user may wonder why `"managed": false` is only appearing for their cloned transforms and not for other other transforms they've created.

If we want to always have a `managed` property in transforms and AD jobs we'll have to add this to the wizards.